src/main/java/Task.java

```diff
@@ -0,0 +1,50 @@
+import java.io.Serializable;
+
+abstract class Task implements Serializable {
+
+    // Attributes
+    protected final String description;
+    protected boolean isDone;
+
+    // Constructor
+
+    /**
+     * Creates a generic task.
+     * @param description description of the task.
+     * @throws EmptyBodyException If description is empty.
+     */
+    public Task(String description) throws EmptyBodyException {
+        if (description.isEmpty()) {
+            throw new EmptyBodyException("description", this.getClass().toString());
+        }
+
+        this.description = description;
+        this.isDone = false;
+    }
+
+    // Methods
+    private String getStatusIcon() {
+        return (this.isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
```

Comment:
I like how you segment the classes with "//Attributes, //Methods, etc." as it makes your code easily readable and clean. Perhaps you could also add some javadoc comments to methods that are not clear in what they do. For instance, for getStatusIcon(), it could be good to include what the output represented.